configure.ac

```diff
@@ -3720,7 +3720,7 @@ AS_VAR_IF([with_system_expat], [yes], [
   LIBEXPAT_LDFLAGS=${LIBEXPAT_LDFLAGS-"-lexpat"}
   LIBEXPAT_INTERNAL=
 ], [
-  LIBEXPAT_CFLAGS="-I\$(srcdir)/Modules/expat"
+  LIBEXPAT_CFLAGS="-I\$(srcdir)/Modules/expat -DXML_GE=1"
```

Comment:
what's that?

Reply:
We have to make a choice, from the Windows builds that are now failing cuz I need to update them as well:

> XML_GE (for general entities) must be defined, non-empty, either 1 or 0 (0 to disable, 1 to enable; 1 is a common default

Reply:
You should probably update the default value in line 3719 as well.

Reply:
> what's that?

@ambv "GE" is short for general entities. The flag is documented in detail at https://libexpat.github.io/doc/api/latest/#XML_GE .

@sethmlarson I would have expected (meaning considered likely) a new line `#define XML_GE 1` to go into [CPython's own `expat_config.h`](https://github.com/python/cpython/blob/main/Modules/expat/expat_config.h). Did you avoid that file on purpose? Its sibling `XML_DTD` seems to live there.

> You should probably update the default value in line 3719 as well.

For anyone else wondering where (or what the line content is), that seems to be: https://github.com/python/cpython/blob/514b1c91b8651e8ab9129a34b7482033d2fd4e5b/configure.ac#L3719

Reply:
@hartwork Thank you for this, I don't know how but my `grep XML_DTD` turned up no results. That's definitely where it should go.